src/app/community/create/create.component.scss

```diff
@@ -0,0 +1,36 @@
+textarea.mat-input-element {
+  resize: none;
+  overflow: auto;
+}
+
+mat-card-content {
+  display: flex;
+  flex-direction: column;
+}
+
+.communityCreationForm {
+  margin: 30px;
+  height: 70vh;
+  min-height: 450px;
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+#communityNameField {
+  border-top: 10px solid transparent;
+}
+
+#communityTitleField {
+  border-top: 10px solid transparent;
+}
+
+#comDescriptionField textarea {
+  height: 30vh;
```

Comment:
I think the height can be much smaller for this, I don't expect people to have really long descriptions.

Reply:
I made it longer because I thought description contains the form's rules and what not. If that's the case, they might slightly format their text and that could increase size? Also because if it's smaller... there's a giant gap between input and button and it looks bad... I can change the card size but then it's a very flat card and it doesn't look great... But I'm down to changing it if you want.